packages/docgen/package.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "docgen",
+	"version": "1.0.0-beta.0",
+	"description": "Autogenerate public API documentation from exports and JSDoc comments.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"jsdoc",
+		"documentation"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/docgen/README.md",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"bin": {
+		"docgen": "./src/cli.js"
+	},
+	"dependencies": {
+		"optionator": "0.8.2"
+	},
+	"devDependencies": {
+		"faucet": "0.0.1",
+		"mdast-util-inject": "1.1.0",
+		"remark": "9.0.0",
+		"remark-parse": "6.0.3",
+		"tape": "4.9.2",
+		"unified": "7.1.0"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"scripts": {
```

Comment:
This is different than all other packages. Unit tests are executed from the root folder for everything else. It should be documented for future maintainers how to run those tests. You also need to keep in mind that to install `devDependencies` in here someone will have to explicitly go to this folder. It won't get installed with `npm i` in the root folder. Only `dependencies` get hoisted.

Reply:
Happy to adapt this to the way other packages are set up. I'm just unfamiliar with the lerna monorepo setup and haven't found any doc about how should we create new packages. Do you have any reference I can check? Or specific steps?

Reply:
Technically speaking we can leave it as is. I just wanted to make sure you are aware of the implications of such setup.

For reference, this is how you can run a script from the package maintained by Lerna:
https://github.com/WordPress/gutenberg/blob/master/package.json#L154
`lerna run build` in particular is the way to run all `build` scripts in packages. See more at https://github.com/lerna/lerna/tree/master/commands/run

Reply:
Fixed what you mentioned:

- tests are now migrated to jest, so they're picked by it and executed as any other WordPress package.
- dependencies are now hoisted.